WordPress/src/main/java/org/wordpress/android/ui/stats/NestedScrollViewExt.java

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.stats;
+
+import android.content.Context;
+import android.support.v4.widget.NestedScrollView;
+import android.util.AttributeSet;
+import android.widget.ScrollView;
+
+public class NestedScrollViewExt extends NestedScrollView {
```

Comment:
Is this class solely for use with stats? If we think it may be used elsewhere, it should be moved to the `ui/widgets` folder.